content/projects/intro-to-database/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Introduction to database
+
+ready: True
+
+## Task
+
+- create a database and name it “mystore”
```

Comment:
Capitalise the beginning of all or none of the sentences, it looks off if all but 2 are capitalised